File: XIVBLM/XIVBLM_py/Update Tables.py
import pandas as pd
import numpy as np

PPSTable = pd.read_csv('Tables/PPSTable.csv')
np.save('data/PPSTable.npy', PPSTable.values)

MainStatValues = pd.read_csv('Tables/MainStatValues.csv', index_col=0).values
np.save('data/MainStatValues.npy', MainStatValues)

SecondaryMinorValues = pd.read_csv('Tables/SecondaryMinorValues.csv', index_col=0).values
np.save('data/SecondaryMinorValues.npy', SecondaryMinorValues)

SecondaryStatCapValues = pd.read_csv('Tables/SecondaryStatCapValues.csv', index_col=0).values
np.save('data/SecondaryStatCapValues.npy', SecondaryStatCapValues)

WDValues = pd.read_csv('Tables/WDValues.csv', index_col=0).values
np.save('data/WDValues.npy', WDValues)

GearType = pd.read_csv('Tables/GearType.csv')
GearType.to_pickle('data/GearType.pkl')

Menu = pd.read_csv('Tables/Menu.csv')
Menu.to_pickle('data/Menu.pkl')

File: XIVBLM/R/Update Tables.R
PPSTable<-read.csv('Tables\\PPSTable.csv')
save(PPSTable,file='data\\PPSTable.rda')

MainStatValues<-as.matrix(read.csv('Tables\\MainStatValues.csv',row.names = 1))
save(MainStatValues,file='data\\MainStatValues.rda')

SecondaryMinorValues<-as.matrix(read.csv('Tables\\SecondaryMinorValues.csv',row.names = 1))
save(SecondaryMinorValues,file='data\\SecondaryMinorValues.rda')

SecondaryStatCapValues<-as.matrix(read.csv('Tables\\SecondaryStatCapValues.csv',row.names = 1))
save(SecondaryStatCapValues,file='data\\SecondaryStatCapValues.rda')

WDValues<-as.matrix(read.csv('Tables\\WDValues.csv',row.names = 1))
save(WDValues,file='data\\WDValues.rda')

GearType<-read.csv('Tables\\GearType.csv')
save(GearType,file='data\\GearType.rda')

Menu<-read.csv('Tables\\Menu.csv')
save(Menu,file='data\\Menu.rda')
